custom_components/wyzeapi/siren.py

```diff
@@ -40,8 +40,8 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry,
     camera_service = await client.camera_service
     sirens = []
     for camera in await camera_service.get_cameras():
-        # The Campan and V2 cameras don't have a siren
-        if camera.product_model not in ["WYZECP1_JEF", "WYZEC1-JZ"]:
+        # The campan (v1 and 2), v2 camera, and video doorbell pro don't have sirens
+        if camera.product_model not in ["WYZECP1_JEF", "WYZEC1-JZ", "GW_BE1"]:
```

Comment:
You mention the campan V2 doesn't have a siren, is it a different model number than `WYZECP1_JEF`? If so it needs to be added to the array.

Reply:
Im not sure, I only have the cam pan v3 to test.  I did not add that check and comment, just updated with another model

Reply:
It looks like the V2 does have a siren, so the comment should be updated.